source/deploy/standalone/dev_ssl.md.erb

```diff
@@ -0,0 +1,845 @@
+---
+title: Using SSL with Passenger in development
+section: walkthroughs
+---
+<%
+locals.merge!(INTEGRATION_MODE_STANDALONE)
+globals.merge!(INTEGRATION_MODE_STANDALONE)
+content_for(:sidebar) do
+    render_partial("../sidebar.html", locals: locals.merge(
+        limit_choices: { "intmode" => [INTEGRATION_MODE_STANDALONE] }
+    ))
+end
+%>
+
+<h1>Using SSL with Passenger in development</h1>
+
+It is a good idea to have your dev env be as close to your production env as possible, without getting in your way. For example setting up ssl in your dev env helps you to fix mixed content warnings so there are no surprises when you move to staging or production, and doesn't train you to click through ssl warnings in your browser.
+
+## Prepare the system
+
+Assumptions: you are using the bash shell, and have admin rights to your computer, and are using Firefox, Chrome, or Safari. Ensure that openssl, passenger, dnsmasq, and optionally nginx or apache are installed.
+
+Follow these instructions to install passenger: https://www.phusionpassenger.com/library/install/apache/install/enterprise/ you may have to adjust the settings to your desired config, for example using passenger standalone or with nginx.
```

Comment:
You can link to that section of the library with `<%= link_to %>`. There are examples elsewhere in the library.

You should mind the capitals, so Passenger Standalone rather than passenger standalone.